maven-slf4j-provider/src/main/java/org/slf4j/impl/MavenSimpleLogger.java

```diff
@@ -30,6 +29,14 @@
  * @since 3.5.0
  */
 public class MavenSimpleLogger extends SimpleLogger {
+
+    private static final String TRACE_RENDERED_LEVEL = builder().trace("TRACE").build();
+    private static final String DEBUG_RENDERED_LEVEL = builder().debug("DEBUG").build();
+    private static final String INFO_RENDERED_LEVEL = builder().info("INFO").build();
+    private static final String WARN_RENDERED_LEVEL =
+            builder().warning("WARNING").build();
+    private static final String ERROR_RENDERED_LEVEL = builder().error("ERROR").build();
```

Comment:
I see your point, I will try to find and propose a better solution a better solution. I would like to avoid recreating those strings on each `MavenSimpleLogger` instances. I know that the usual use case is that there is only on instance of `MavenSimpleLogger`, the root logger, but it is also possible to configure loggers per component or class with more efforts.